src/main/java/uk/ac/cam/cl/dtg/segue/api/SegueDefaultFacade.java

```diff
@@ -89,6 +89,7 @@ public SegueDefaultFacade(final PropertiesLoader properties, final ContentMapper
     @Path("/")
     @Produces(MediaType.TEXT_HTML)
     @Cache
+    @Deprecated
```

Comment:
I think originally this was here so that when someone went to /api they gout the swagger page which out having to type anything else. Does this now work as part of the swagger upgrade?